RWFramework/RWFramework/Playlist/Playlist.swift

```diff
@@ -319,7 +319,7 @@ extension Playlist {
     }
     
     /// Periodically check for newly published assets
-    @objc private func heartbeat() {
+    @objc public func refreshAssetPool() {
```

Comment:
This can be `internal`, as we shouldn't be using it outside the library I think.
```suggestion
    @objc internal func refreshAssetPool() {
```

Reply:
clearly I need to learn the differences between these distinctions. I agree it should be as restricted as possible, so if `internal` works for calling it from all the places we need to call it, that's great.